xchange-kraken/src/main/java/info/bitrich/xchangestream/kraken/KrakenStreamingService.java

```diff
@@ -27,18 +27,20 @@
  * @author makarid, pchertalev
  */
 public class KrakenStreamingService extends JsonNettyStreamingService {
```

Comment:
Looking at this code, there seems to be so many `if.. else` sections, I wonder if it might be more clearly implemented as two separate classes, one private and one public. Thoughts?

Reply:
It looks the next version  (beta-ws)  will merge the two back together (private and public), so might be worth waiting before creating new class for private?

Reply:
Fair enough.